docs/collecting-data/collecting-from-own-applications/python-tracker/previous_versions/python-v0-15/upgrading/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Upgrading to Newer Versions"
+date: "2023-01-25"
+sidebar_position: 100
+---
+
+This page gives instructions to upgrading to newer versions of the Python Tracker
+
+## Upgrading to v0.13.0
```

Comment:
Upgrading to 0.13.0 inside a 0.15 version page? 🤔

Reply:
Just a draft atm! Currently re-writing them all for V1

Reply:
This comment is still relevant :D